src/Windowing/Silk.NET.Windowing.Common/Window.cs

```diff
@@ -0,0 +1,197 @@
+// This file is part of Silk.NET.
+// 
+// You may modify and distribute Silk.NET under the terms
+// of the MIT license. See the LICENSE file for details.
+
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using Silk.NET.Windowing.Internals;
+
+namespace Silk.NET.Windowing
+{
+    /// <summary>
+    /// Convenience wrapper for easily creating a Silk.NET window.
+    /// </summary>
+    public static class Window
+    {
+        public static IReadOnlyList<IWindowPlatform> Platforms { get; } = new List<IWindowPlatform>();
+        internal static Exception NoPlatformException => new Exception
```

Comment:
https://docs.ultz.co.uk/Silk.NET/windowing/troubleshooting.html mentions `NotSupportedException` this throws `Exception` I'd suggest `PlatformNotSupportedException`

Reply:
Ah nice catch. Will update both.
```suggestion
        internal static Exception NoPlatformException => new PlatformNotSupportedException
```